tvm/python/tvm/topi/adreno/conv2d_nhwc.py

```diff
@@ -281,17 +333,31 @@ def copy_to_texture(stage):
 
     s[latest].compute_root()
 
-    axes = s[latest].op.axis
-    fused = s[latest].fuse(*axes[:-1])
     N, OH, OW, OC = get_const_tuple(latest.shape)
 
-    if OC < 32:
-        block, thread = s[latest].split(fused, factor=32)
-        s[latest].bind(block, te.thread_axis("blockIdx.x"))
-        s[latest].bind(thread, te.thread_axis("threadIdx.x"))
+    if OC % 4 == 0:
+        n, oh, ow, oc = s[latest].op.axis
+        occ, ocb = s[latest].split(oc, factor=4)
+        s[latest].reorder(n, oh, ow, occ, ocb)
+        s[latest].vectorize(ocb)
```

Comment:
Just out of curiosity, does this vectorization apply to just rewrites of conv operation or of any operation fused into conv operation as well?

For e.g. if relu is fused into conv (conv+relu), does this vectorization apply to relu rewrites?

Reply:
it is applied for all post-ops (been fused to one op and scheduled in this place)